core/example/03_struct_of_arrays/soa_example.cpp

```diff
@@ -0,0 +1,142 @@
+/****************************************************************************
+ * Copyright (c) 2018 by the Cabana authors                                 *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <Cabana_Core.hpp>
+
+#include <iostream>
+
+//---------------------------------------------------------------------------//
+// SoA example.
+//---------------------------------------------------------------------------//
+void soaExample()
+{
+    /*
+      Cabana structs-of-arrays (SoAs) are groups of Tuples with the data
+      specified in an order such that the tuple index is stride-1 in
```

Comment:
"tuple index is stride-1..." may be "tuple's member array index is stride 1..."?